modules/dxp/apps/commerce-avalara/commerce-avalara-connector/bnd.bnd

```diff
@@ -0,0 +1,4 @@
+Bundle-Name: Liferay Commerce Avalara Connector
+Bundle-SymbolicName: com.liferay.commerce.avalara.connector
+Bundle-Version: 1.0.0
+Export-Package: com.liferay.commerce.avalara.connector
```

Comment:
@CalvinKeum I ran `ant format-source-current-branch` in portal-impl and looks like this needs to be added.

`Liferay-Enterprise-App: dxp.only=true;product.id=9a473157-06a6-44b6-b017-a360ffaf5f38`

I think that's the product.id for Commerce modules and `dxp.only=true` is just to mark this app as dxp only.

Reply:
Got it, will add, thanks Danny.